marlowe-runtime/app/Main.hs

```diff
@@ -1,8 +1,4 @@
 module Main where
 
-import qualified MyLib (someFunc)
-
 main :: IO ()
-main = do
-  putStrLn "Hello, Haskell!"
-  MyLib.someFunc
+main = undefined
```

Comment:
So the plan is to put APIs in `marlowe-runtime`. Will the `main` here evolve into a CLI entry point? For clients, for control, or both?

Reply:
This `main` will eventually evolve into the CLI for the runtime daemon as a singleton process.